Controller/ElFinderController.php

```diff
@@ -184,8 +185,12 @@ public function loadAction(Request $request, $instance, $homeFolder)
     {
         $httpKernel = $this->get('http_kernel');
 
-        $preExecutionEvent = new ElFinderPreExecutionEvent($request, $httpKernel, $instance, $homeFolder);
-        $this->get('event_dispatcher')->dispatch(ElFinderEvents::PRE_EXECUTION, $preExecutionEvent);
+        try {
+            $preExecutionEvent = new ElFinderPreExecutionEvent($request, $httpKernel, $instance, $homeFolder);
```

Comment:
i think rather than doing a try catch here you should probably do it on your service listener, maybe here too is ok too but that way you can customize the exception handling